src/Microsoft.Azure.NotificationHubs/NamespaceManagerSettings.cs

```diff
@@ -0,0 +1,50 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// Licensed under the MIT License. See License.txt in the project root for 
+// license information.
+//------------------------------------------------------------
+
+using System;
+using System.Net.Http;
+using Microsoft.Azure.NotificationHubs.Auth;
+using Polly.Retry;
+
+namespace Microsoft.Azure.NotificationHubs
+{
+    internal sealed class NamespaceManagerSettings
+    {
+        private AsyncRetryPolicy<HttpResponseMessage> retryPolicy;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NamespaceManagerSettings"/> class.
+        /// </summary>
+        public NamespaceManagerSettings()
```

Comment:
Add another ctor() with TokenProvider and RetryPolicy as args. This would be if the customer wants to use its own retry policy.